settings/config.settings.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+// Ensure the appropriate config split is enabled.
+$config['config_split.config_split.local']['status'] = FALSE;
+$config['config_split.config_split.dev']['status'] = FALSE;
+$config['config_split.config_split.stage']['status'] = FALSE;
+$config['config_split.config_split.prod']['status'] = FALSE;
+$config['config_split.config_split.ci']['status'] = FALSE;
```

Comment:
Also see my notes-in-progress here: https://github.com/acquia/blt/issues/965#issuecomment-288171604